cogs/tickets.py

```diff
@@ -17,27 +31,33 @@
 class Tickets(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
-        self.db  = bot.db
+        self.db = bot.db
```

Comment:
Line the equal signs up, `self.db` should also be above `self.bot` due to it's length.